Domain Connect Spec Draft.adoc

```diff
@@ -1131,23 +1131,30 @@ defined.
 call was successful and the template applied. Note that any 200 level
 code should be considered a success.
 
+|*Bad Request*
+|400
+|A response of a 400 indicates that the server cannot process the request because it
+was malformed or had errors. This response code is intended for programming errors.
+
+
 |*Unauthorized*
 |401
 |A response of a 401 indicates that caller is not
 authorized to make this call. This can be because the token was revoked,
 or other access issues.
 
-|*Error*
-|400, 404, 422
-|This indicates something wrong with the request
-itself, such as bad parameters.
-
 |*Failed*

```

Comment:
Do we keep it "Failed" or rather "Conflict" ?

Reply:
409 should probably be "Conflict", matches HttpStatusCode.Conflict.
Can you also detail 403 Forbidden? We get that from GoDaddy/SecureServer, at least our action is to prompt user to re-authenticate.